### DISCUSSION TITLE
T-Pot to AbuseIPDB?

### DISCUSSION THREAD

**Discussion by c-sakel** [telekom-security/tpotce#1275] [General]
Hello, I would like to export with T-Pot attack to AbuseIPDB and similar Spam Pages.
Does anyone have an idea or know how to export them directly through T-Pot if necessary?

**Comment by CodeMeZone** [comment]
I know this is a little old, but I'm on the same page, any help here about sending to abuseipdb

**Comment by sefinek** [comment]
I've developed an integration between T-Pot and AbuseIPDB. It currently supports Cowrie, Dionaea, and Honeytrap. Support for more honeypots is coming soon.

https://github.com/sefinek/T-Pot-To-AbuseIPDB
I hope someone finds it useful!

![WindowsTerminal_q2AjzrtiPgSf](https://github.com/user-attachments/assets/656e07a0-fa32-4d62-8eb9-c5eae757872f)

**Comment by dmille6** [comment]
I use a python script that runs on the hive server (so I dont have to open any ports)
it runs every 15 min through a cron job. 

python script queries ES index, then runs through 15 min of data, parses, normalizes, and submits the stuff to Alienvault OTX. you could do the same for abuseIPDB or anything else like that. 

thats my approach. another approach (I'm working on) is building a plugin/connector for OpenCTI that will pull/injest data from TPOT. OpenCTI has connectors for all the data sharing platforms like abuseIPDB

I hope this helps
